### PR TITLE
switched from uws to ws

### PR DESCRIPTION
**Pull request by etrousset**
## What does this PR do ?
Remove dependence to uws and switched to ws. 

This should allow running Kuzzle on NodeJS 10

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -120,6 +120,26 @@ jobs:
       script:
         - docker-compose -f .ci/test.yml run kuzzle
 
+# ---------------------------------------
+# Node 10 - x64
+# ---------------------------------------
+    - stage: Functional tests
+      name: "Functionnal Tests - amd64 - Node v10"
```
Comment (xbill82) [inline .travis.yml]:
```suggestion
      name: "Functional Tests - amd64 - Node v10"
```

**Comment by etrousset** [comment]
@benoitvidis : As I expected the two packages had the same interface, I just replaced package `uws` with `ws` and launched tests. Has nothing seemed to be broken, I thought it was ok.

But actually, you are right the `require('uws') ` should have failed.... so it looks like the tests doesn't really test the websocket protocol?

**Comment by scottinet** [comment]
@Aschen > Did you try it? From what I know, the node bindings are unmaintained and do not work with node.js 10

**Comment by Aschen** [comment]
Hmm I just tested it and it seems to be a different library.
```
console.log('uWebSockets.js');
{ App: [Function],
  SSLApp: [Function],
  nextTick: [Function],
  processNextTickQueue: [Function],
  print: [Function],
  us_listen_socket_close: [Function] }
```
Nevertheless, I still think that we can use Discord fork or use our own fork based on their till the API didn't change.

**Comment by scottinet** [comment]
obsoleted by https://github.com/kuzzleio/kuzzle/pull/1301

**Pull request closed without merging**